.github/workflows/playwright.yml

```diff
@@ -0,0 +1,45 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: gcs
+    
+    steps:
+    - uses: actions/checkout@v4
+    
+    - uses: actions/setup-node@v4
+      with:
+        node-version: lts/*
+    
+    - name: Install dependencies
+      working-directory: gcs
+      run: npm install -g yarn && yarn
+    
+    - name: Install Playwright Browsers
+      working-directory: gcs
+      run: yarn playwright install --with-deps
+
+    - name: Run FGCS frontend
+      run: yarn dev:test &
+    
+    - name: Run Playwright tests
+      uses: coactions/setup-xvfb@v1
+      with:
+        working-directory: gcs
+        run: yarn playwright test
+        # run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- yarn playwright test
```

Comment:
what is xvfb-run?

Reply:
its a way of running x-servers without a monitor, we don't need it - i'll get rid of it!